checkov/common/util/env_vars_config.py

```diff
@@ -78,5 +78,8 @@ def __init__(self) -> None:
         # need to fix usage, because the env var value is set inside the code
         self.GITHUB_CONF_DIR_PATH = os.getenv("CKV_GITHUB_CONF_DIR_PATH")
 
+        # Allows using kustomize commands to directly edit the user's kustomization.yaml configurations
+        self.ALLOW_KUSTOMIZE_FILE_EDITS = convert_str_to_bool(os.getenv("ALLOW_KUSTOMIZE_FILE_EDITS", False))
```

Comment:
```suggestion
        self.ALLOW_KUSTOMIZE_FILE_EDITS = convert_str_to_bool(os.getenv("CHECKOV_ALLOW_KUSTOMIZE_FILE_EDITS", False))
```
please prefix it with `CHECKOV_` to make it clear, it is an internal env var.

Reply:
done